src/test/java/com/clouway/cuse/SearchEngineContractTest.java

```diff
@@ -979,6 +979,40 @@ public void searchEmptyString() throws Exception {
     assertThat(result.size(), is(3));
   }
 
+  @Test
+  public void registerDocumentsInBulkChunks() throws Exception {
+    List<Object> employeeList = generateGiantListOfEmployeesNamedFooBar(402);
+    storeAllEmployee(employeeList);
+    store(new User(2l));
```

Comment:
It can be difficult to distinguish the `l` from a `1` in a `long` value. Better to use uppercase `L`.